### PR TITLE
doc: update docs

### PR DESCRIPTION
**Pull request by karasusan**
## TODO
Translate in English

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -7,17 +7,18 @@ This solution's streaming technology takes advantage of [WebRTC](https://webrtc.
 
 <img src="com.unity.renderstreaming/Documentation~/images/browser_hdrpscene.png" width=500 align=center>
 
-This repository manages 2 packages. Details can be found in the `README.md` for each package.
+This repository manages a 1 package and 2 templates. Details can be found in the `README.md` for each package.
```
Comment (sindharta) [inline README.md:10]: ~a~

File: com.unity.renderstreaming/Documentation~/en/faq.md [hunk 2]
```diff
@@ -4,7 +4,9 @@
 
 ### Video stream not showing in browser
 
-The browser version might be too old. Make sure that the latest version is installed.
+The browser version might be too old. Make sure that the latest version is installed. Or firewall settings can be caused by the issue, please try once to turn off it.
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/en/faq.md:7]: -> Or firewall settings can be caused by the issue, please try once to turn off it.

Or firewall settings can also cause the issue, please try to turn it off once.

File: com.unity.renderstreaming/Documentation~/en/faq.md [hunk 3]
```diff
@@ -4,7 +4,9 @@
...
+
+Please try to check no error log on the browser console and Unity debug console.
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/en/faq.md:9]: Please check if there is any error log on the browser console and Unity debug console.

File: com.unity.renderstreaming/Documentation~/en/faq.md [hunk 4]
```diff
@@ -49,15 +51,15 @@ Currently, version 1.0 does not support simulcast.
 
 ### Can you broadcast the feed from multiple cameras?
 
-Currently, version 1.0 cannot.
+It is available. If you use the hardware encoder, make sure the limitation of the number of cameras using at the same time. See the NVIDIA [document]](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix).
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/en/faq.md:54]: Yes. If you use the hardware encoder, make sure to check the limitation of the number of cameras used at the same time. See the NVIDIA [document]](https://developer.nvidia.com/video-encode-decode-gpu-support-matrix).

File: com.unity.renderstreaming/Documentation~/index.md [hunk 5]
```diff
@@ -19,12 +19,12 @@ This version of Render Streaming is compatible with the following versions of th
 - **Unity 2019.3**
 
 > [!NOTE]
-> Because this template depends on `HDRP 7.1`, it does not work correctly older Unity versions.
+> Because this template depends on `HDRP 7.3`, it does not work correctly older than `Unity 2019.3`.
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/index.md:22]: older **versions** than `Unity 2019.3`.

File: com.unity.renderstreaming/Documentation~/en/faq.md [hunk 6]
```diff
@@ -49,15 +51,15 @@ Currently, version 1.0 does not support simulcast.
...
 
 ### How do you increase the video quality?
 
-Video quality depends on the quality of the network. 
+You can set the resolution of the frame on the `CameraStreamer` inspector. Please make sure video quality depends on the quality of the network.
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/en/faq.md:58]: ~Please make sure~ video quality depends on the quality of the network. 

Keep in mind that video quality depends on the quality of the network.

File: com.unity.renderstreaming/Documentation~/en/https.md [hunk 7]
```diff
@@ -4,9 +4,11 @@
 
 ### Windows
 
+This document is written about setting on Windows.
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/en/https.md:7]: This document is written assuming that the OS is Windows.

File: com.unity.renderstreaming/Documentation~/en/faq.md [hunk 8]
```diff
@@ -49,15 +51,15 @@ Currently, version 1.0 does not support simulcast.
...
 
 ### How do you increase the video quality?
 
-Video quality depends on the quality of the network. 
+You can set the resolution of the frame on the `CameraStreamer` inspector. Please make sure video quality depends on the quality of the network. 
 
 ### Is Realtime-Raytracing supported? 
 
-Currently, version 1.2 cannot.
+It is available. Please see the detail on the HDRP [document](https://docs.unity3d.com/Packages/com.unity.render-pipelines.high-definition@7.1/manual/Ray-Tracing-Getting-Started.html).
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/en/faq.md:62]: ~It is available~

Yes.

File: com.unity.renderstreaming/Documentation~/en/components.md [hunk 9]
```diff
@@ -19,9 +21,27 @@
...
 | **Hardware Encoder Support** | Enable the checkbox to make this video encoding replace values that use a [hardware encoder](../index.md#hardware-encoder) | Enabled                         |
 | **Array Button Click Event** | Register browser button click events                                                                                       |                                 |
 | *Element Id*                 | Set an ID that is sent from browsers                                                                                       |                                 |
-| *Click*                      | Set button click event                                                                                                     |                                 |
\ No newline at end of file
+| *Click*                      | Set button click event                                                                                                     |                                 |
+
+## `CameraStreamer` Component
```
Comment (sindharta) [inline com.unity.renderstreaming/Documentation~/en/components.md:29]: This is supposed to be an English page, right ?

Comment (karasusan) [reply com.unity.renderstreaming/Documentation~/en/components.md:29]: Yes, I will request Chris translating it in English

Comment (sindharta) [reply com.unity.renderstreaming/Documentation~/en/components.md:29]: Ok

File: com.unity.renderstreaming/Documentation~/en/faq.md (hunk 8 above)
Comment (sindharta) [reply com.unity.renderstreaming/Documentation~/en/faq.md]: This one is missing from the previous fix.

Comment (karasusan) [reply com.unity.renderstreaming/Documentation~/en/faq.md]: Fixed it

**Pull request merged**